# Service that can be used to get a valid random pose for the robot

---
bool success
string[] joint_names
float64[] joint_positions